src/app/organizations/settings/subscription-hidden.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component, OnDestroy, OnInit } from "@angular/core";
+
+import { PlatformUtilsService } from "jslib-common/abstractions/platformUtils.service";
+import { ThemeType } from "jslib-common/enums/themeType";
+
+interface Colors {
+  shapeColor: string;
+  lineColor: string;
+}
+@Component({
+  selector: "app-subscription-hidden",
+  templateUrl: "./subscription-hidden.svg",
+})
+export class SubscriptionHiddenComponent implements OnInit, OnDestroy {
+  constructor(private platformUtilsService: PlatformUtilsService) {}
+
+  themes: Partial<Record<ThemeType, { shape: string; line: string }>> = {
+    [ThemeType.Light]: {
+      shape: "#E1E5EE",
+      line: "#2F343D",
+    },
+    [ThemeType.Dark]: {
+      shape: "#2F343D",
+      line: "#E1E5EE",
+    },
+  };
+  colors = this.themes[ThemeType.Light];
```

Comment:
Could we use css variables here?

Reply:
Yes, this works! I'll raise it with design to ensure we can use standardised colours.